### ISSUE TITLE
Support for seeding previously downloaded items

### ISSUE THREAD

**Issue by iadj** [mafintosh/playback#43]
A growing problem with streaming apps is that there's a decreasing amount of long term seeders.

A solution would be the option to specify a download/stream folder that doesn't automatically clear on close. When the player opens, it automatically starts seeding the files inside. This way every time you stream content or have the player running you'll give back to the community by seeding your streamed content. This could be an optional feature which people can choose to turn on.

**Comment by mafintosh** [comment]
Good idea. I'm thinking we could start persisting the playlist and you'd just seed everything thats currently in your playlist.
